src/main/java/net/foxdenstudio/sponge/foxguard/plugin/handler/WelcomeHandler.java

```diff
@@ -66,16 +107,33 @@ public WelcomeHandler(String name, boolean isEnabled, int priority) {
 
     @Override
     public ProcessResult modify(CommandSource source, String arguments) throws CommandException {
-        return null;
+        AdvCmdParser.ParseResult parse = AdvCmdParser.builder().arguments(arguments).limit(2).parse();
+
+        if (parse.args.length < 1) return ProcessResult.of(false, Text.of("Must specify a command!"));
+
+        if(parse.args.length < 2) return ProcessResult.success();
+
+        if(parse.args[0].equals("enter")) enterTemplate = fromString(parse.args[1]);
+        else if(parse.args[0].equals("exit")) exitTemplate = fromString(parse.args[1]);
+        else ProcessResult.of(false, "Invalid Command.");
+
+        return ProcessResult.success();
     }
 
     @Override
     public List<String> modifySuggestions(CommandSource source, String arguments, @Nullable Location<World> targetPosition) throws CommandException {
-        return null;
+        return Lists.newArrayList("enter", "exit");
```

Comment:
So you need to be doing a lot more here than just returning a list of options. This code isn't magic. The list you send is the exact list that is displayed to the user.

If a user types "cake" and then hits tab, the word "cake" would be entirely replaced by the word "enter".

For reference, it should look something like this:

https://github.com/FoxDenStudio/FoxCore/blob/api5/src/main/java/net/foxdenstudio/sponge/foxcore/plugin/command/CommandHUD.java#L92-L105

It seems roundabout, but it ensures that it won't replace anything it's not supposed to.

You could probably just copy and paste the code, and replace the literal words.